packages/medusa-plugin-auth/README.md

```diff
@@ -276,3 +276,55 @@ Coming soon
 ### Microsoft
 
 Coming soon
+
+### Firebase
+
+> By default, the admin only allow to authenticate while the store create a new user of it does not exist yet.
+> This behaviour can be changed and customised by specifying a custom `verifyCallback` in the configuration.
+
+Then, in your medusa config plugins collection you can add the following configuration and update it according to your requirements ([full configuration here](https://github.com/adrien2p/medusa-plugins/tree/main/packages/medusa-plugin-auth/src/auth-strategies/firebase/types.ts))
+
+```ts
+{
+    resolve: "medusa-plugin-auth",
+    options: {
+        // Enable Firebase auth
+        firebase: {
+            // ----------- REQUIRED -----------
+            credentialJsonPath: "__PATH_TO_JSON_CREDENTIALS__",
+            // Enable Firebase auth for the admin domain
+            admin: {
+                // ----------- OPTIONAL -----------
+                // authPath: '/admin/auth/firebase',
+                // expiresIn: 24 * 60 * 60 * 1000
+            },
+            // Enable Firebase auth for the store domain
+            store: {
+                // ----------- OPTIONAL -----------
+                // authPath: '/admin/auth/firebase',
+                // expiresIn: 24 * 60 * 60 * 1000
+            }
+        }
+        // ...
+        // ... Other authentication provider options
+        // ...
+    }
+}
+```
+
+Now you can authenticate with Firebase ID token by calling the following endpoint
+
+```ts
+const firebaseLogin = async (token: string) => {
+    await fetch('http://localhost:9000/store/auth/firebase', {
```

Comment:
suggestion: Maybe use `${medusa_url}/${firebase_authPath}`  like the other description?